lib/job/arUpdateEsIoDocumentsJob.class.php

```diff
@@ -56,36 +56,44 @@ public function runJob($parameters)
     $this->job->addNoteText($message);
     $this->info($message);
 
+    $count = 0;
     foreach ($parameters['ioIds'] as $id)
     {
+      $count++;
+
       if (null === $object = QubitInformationObject::getById($id))
       {
         $this->info($this->i18n->__('Invalid archival description id: %1', array('%1' => $id)));
 
         continue;
       }
 
-      $title = $object->getTitle(array('cultureFallback' => true));
-
       if ($parameters['updateIos'] && $parameters['updateDescendants'])
       {
         arElasticSearchInformationObject::update($object, array('updateDescendants' => true));
-        $message = $this->i18n->__('Updating "%1" and descendants.', array('%1' => $title));
+        $message = $this->i18n->__('Updated %1 description(s) and their descendants.', array('%1' => $count));
       }
       elseif ($parameters['updateIos'])
       {
         arElasticSearchInformationObject::update($object);
-        $message = $this->i18n->__('Updating "%1".', array('%1' => $title));
+        $message = $this->i18n->__('Updated %1 description(s).', array('%1' => $count));
       }
       else
       {
         arElasticSearchInformationObject::updateDescendants($object);
-        $message = $this->i18n->__('Updating descendants of "%1".', array('%1' => $title));
+        $message = $this->i18n->__('Updating descendant of %1 description(s).', array('%1' => $count));
       }
 
-      $this->info($message);
+      // Status update every 100 descriptions
+      if (0 == $count % 100)
+      {
+        $this->info($message);
+      }
     }
 
+    // Final status update
+    $this->info($message);
```

Comment:
This message will be printed twice if the number of objects being indexed is a multiple of 100.

Reply:
Thanks @sbreker, I've fixed this.